website/docs/index.mdx

```diff
@@ -178,6 +179,12 @@ If you have any questions, please reach out to our awesome community on [Slack](
               description="How to package and release your own custom library"
               to="/docs/extending_robot_framework/custom-libraries/releasing_your_own_libraries"
             />
+            <Card
+              title="Python Library Template"
+              icon={<CopyIcon/>}
+              description="Create a Python library using an officially supported GitHub repository template"
+              to="https://github.com/fkromer/robotframework-dosomething"

```

Comment:
i have no objection but saying "officially supported" and then linking into personal repository feels a bit off.